lib/fluent/plugin/in_rdkafka_group.rb

```diff
@@ -236,10 +174,42 @@ def reconnect_consumer
     end
   end
 
+  class Batch
+    attr_reader :topic
+    attr_reader :messages
+
+    def initialize(topic)
+      @topic = topic
+      @messages = []
+    end
+  end
+
+  def each_batch(&block)
```

Comment:
Please add some comments on the purpose of new classes / methods where non-trivial.

Reply:
Good point, added some comments.